docs/src/main/sphinx/release/release-466.md

```diff
@@ -0,0 +1,72 @@
+# Release 466 (27 Nov 2024)
+
+## General
+
+* Add support for changing the type of nested fields in columns using the map
+  type. ({issue}`24248`)
+* Remove the requirement for a Python runtime on Trino cluster nodes. ({issue}`24271`)
+* Improve performance of queries involving `GROUP BY` and joins. ({issue}`23812`)
+* Improve client protocol throughput by introducing the [spooling
+  protocol](protocol-spooling). ({issue}`24214`)
+
+## Security
+
+* Add support for [data access control with Apache
+  Ranger](/security/apache-ranger-access-control), including support for
+  column masking, row filtering, and audit logging. ({issue}`22675`)
+
+## JDBC driver
+
+* Improve throughput by automatically using the [spooling
+  protocol](jdbc-spooling-protocol) when it is configured on the Trino cluster,
+  and add the parameter `encoding` to optionally set the preferred encoding from
+  the JDBC driver. ({issue}`24214`)
+* Improve decompression performance when running the client with Java 22 or
+  newer. ({issue}`24263`)
+* Improve performance of querying catalog metadata from `system.jdbc.tables`.
```

Comment:
Instead of mentioning the underlying table (which is an implementation detail of the driver), describe which API is now faster.